code/upaya/Gemfile

```diff
@@ -2,9 +2,9 @@ source 'https://rubygems.org'
 ruby '2.3.0'
 
 gem 'rails', '~> 4.2.6'
-
 gem 'attr_encrypted'
 gem 'autoprefixer-rails', '~> 5.2'
+gem 'bullet'
```

Comment:
I would prefer not to merge this with a TODO. Bullet should not be deployed to production. During deployment, only production gems should be installed, so I'm not sure why Bullet is causing an error. Perhaps the buildpack is not setting the proper flags to Bundler? It should be issuing this command: `bundle install --deployment --without deploy development test`